### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by suzubara**
* Update Twitter links with Bluesky
* Update Slack link with Discord
* Update organizers info to be current

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![Eleventy Build](https://github.com/austinjavascript/austinjavascript.com/workflows/Eleventy%20Build/badge.svg)
 
-website for the [@AustinJS](https://twitter.com/AustinJS) meetup: http://austinjavascript.com
+website for the [@AustinJS](https://bsky.app/profile/austinjavascript.com) meetup: http://austinjavascript.com
```
Comment (joahg) [inline README.md:3]: nit: should change the handle to `@austinjavascript.com`

**Review by joahg — APPROVED** [review]
thank you!!

File: README.md (hunk 1 above)
Comment (suzubara) [inline README.md:3]:
```suggestion
website for the [@austinjavascript.com](https://bsky.app/profile/austinjavascript.com) meetup: http://austinjavascript.com
```

File: _data/people.json [hunk 2]
```diff
@@ -108,5 +108,15 @@
...
+  },
+  "srozier": {
+    "name": "Suzanne Rozier",
+    "avatar": "https://avatars.githubusercontent.com/u/2723066?v=4",
+    "email": null,
+    "homepage": null,
+    "twitter": null,
+    "github": "suzubara",
+    "linkedin": null,
+    "status": "active"
```
Comment (suzubara) [inline _data/people.json:120]:
```suggestion
    "status": "active",
    },
    "cking": {
    "name": "Collier King",
    "avatar": "https://avatars.githubusercontent.com/u/10591022?v=4",
    "email": null,
    "homepage": null,
    "twitter": null,
    "github": "collierking",
    "linkedin": null,
    "status": "active"
```

**Pull request merged**